app/app/tests/api/api_v1/test_users.py

```diff
@@ -135,3 +135,54 @@ def test_update_profile_picture_superuser(
     assert r.status_code == 200
     assert updated_user['profile_picture']
     assert isfile(f"profile_pictures/{updated_user['profile_picture']}")
+
+
+def test_update_profile_picture_superuser_pdf(
+    client: TestClient, superuser_token_headers: Dict[str, str], db: Session
+) -> None:
+    user = create_random_user(db, type="student")
+    response = client.get("https://raw.githubusercontent.com/mGalarnyk/datasciencecoursera/master/HelloWorld.md")
```

Comment:
Hmm, no please, take something from media.rev-amp.tech or files.rev-amp.tech (basically something we have control of, as opposed to a random file from an external repo)

Reply:
Also method same says PDF, file extension and mimetype are markdown?

Reply:
And this reminds me, how about some tests uploading a text file + image/png, and a png with text/plain or something

Reply:
eeeeeeeek, that would cause errors for sure. so I'll have to figure out type of file from `.content`?

Reply:
not a priority